pkg/network/route_cache.go

```diff
@@ -349,3 +355,19 @@ func (n *netlinkRouter) getInterface(srcAddress util.Address, srcIP net.IP, netn
 	routeCacheTelemetry.ifCacheSize.Inc()
 	return iff
 }
+
+// Adds error tag and increments counter
+func CounterIncWithTag(counter telemetry.Counter, err error) (errno syscall.Errno, ok bool) {
+	errno, ok = err.(syscall.Errno)
```

Comment:
You can make this more concise:
```go
if errno, ok = err.(syscall.Errno); ok {
  	if tag := unix.ErrnoName(errno); tag != "" {
		counter.Inc(tag)
		return
	}
}

counter.Inc()
return
```

Reply:
I think something's weird with my editor... I had already changed that